pkg/virt-launcher/virtwrap/live-migration-target.go

```diff
@@ -171,14 +170,6 @@ func (l *LibvirtDomainManager) prepareMigrationTarget(
 	if err != nil {
 		return err
 	}
-	// TODO this should probably a OnPrepareMigration hook or something.
-	// Right now we need to call OnDefineDomain, so that additional setup, which might be done
-	// by the hook can also be done for the new target pod
-	hooksManager := hooks.GetManager()
-	_, err = hooksManager.OnDefineDomain(&domain.Spec, vmi)
```

Comment:
You basically remove a hook point where a preparation on the pod could have been done to accept the new domain.

The hook point is not only about changing the domain-xml, it is also about doing other preparations based on the VMI and domain.
This may break backward compatibility.

Reply:
But this is still to be called later on, as I mentioned in the commit log. I'm not sure we are breaking compatibility either, no migration tests have failed so far... which it doesn't mean I haven't broken something with this patch.

I'm not sure. By just reading the code, this looks out of place.

Reply:
I'll revert it, just to be safe. Sadly, this was one of the changes I really wanted in because I plan to change `OnDefineDomain` later on and I'll need to replicate it here.... but let's do what is right.

Thanks again for the review.